src/components/bs5/footer/footer.hbs

```diff
@@ -1,136 +1,111 @@
-                                    fill="currentColor"></path>
-                            </svg>
-                            <span class="nav-link-label">{{ followlinks.instagram.label }}</span>
-                        </a>
-                        {{/if}}
+            {{!-- Section 3: Feedback form area --}}
+            {{#if feedbackFormEnabled}}
+            <div class="col-xs-12 col-md-12 col-lg-3">
+                <div class="footer-column {{#if feedbackForm.hasBorderColumn}}border-column{{/if}}">
+                    <div class="row">
+                        <div class="col-xs-12 col-md-6 col-lg-12">
+                            <h3 class="footer-heading">{{feedbackForm.title}}</h3>
+                            <p>{{feedbackForm.content}}</p>
+                        </div>
+                        <div class="col-xs-12 col-md-6 col-lg-12">
+                            <div id="qg-feedback-toggle" class="qg-footer-feedback-wrap">
+                                <a id="btn-footer-feedback" class="btn btn-global-secondary qg-feedback-toggle collapsed" data-bs-toggle="collapse" href="#qg-footer-feedback" role="button" aria-expanded="false" aria-controls="qg-footer-feedback" data-analytics-link-group="qg-feedback">
+                                    {{feedbackForm.btnTitle}}
+                                </a>
+                                <div id="qg-footer-feedback" class="qg-footer-feedback__v2 collapse">
+                                    <script>
```

Comment:
due to CSP security and inlining of scripts, can all inline script elements be moved to fooder.functions.js.

Its far easier to have it in a js file than inline on a security point of view. see https://content-security-policy.com/examples/allow-inline-script/